sys/net/gnrc/netreg/gnrc_netreg.c

```diff
@@ -143,17 +144,31 @@ gnrc_pktsnip_t *gnrc_netreg_hdr_build(gnrc_nettype_t type, gnrc_pktsnip_t *paylo
 #ifdef MODULE_GNRC_IPV6
 
         case GNRC_NETTYPE_IPV6:
-            return gnrc_ipv6_hdr_build(payload, src, src_len, dst, dst_len);
+            assert(src_len == sizeof(ipv6_addr_t));
+            assert(dst_len == sizeof(ipv6_addr_t));
```

Comment:
Hm, wondering about how to document this, I begin to wonder if we need this function at all. What is the purpose of this function?